examples/manipulating_visualizing/plot_prob_atlas.py

```diff
@@ -0,0 +1,60 @@
+"""
+Visualizing 4D atlas maps onto the anatomical image
+===========================================================
+
+Basically this example gives an idea on how to visualize the atlas maps which are
+4D using three different display types, for instance if you choose
+
+1. "contours", which means maps or ROIs are shown as contours delineated by \
+    colored lines.
+
+2. "filled_contours", maps are shown as contours same as above but with \
+    fillings inside the contours.
+
+3. "continuous", maps are shown as just color overlays.
+
+This function can display each map with each different color which are picked
```

Comment:
"This function"? Which function? I would say:

"The :func:`nilearn.plotting.plot...` function displays..."